File: problems/2194_cells_in_a_range_on_an_excel_sheet/Solution.java
import java.util.*;

class Solution {
    public List<String> cellsInRange(String s) {
        List<String> result = new ArrayList<>();
        for (char c = s.charAt(0); c <= s.charAt(3); c++) {
            for (char r = s.charAt(1); r <= s.charAt(4); r++) {
                result.add("" + c + r);
            }
        }
        return result;
    }
}

File: problems/2194_cells_in_a_range_on_an_excel_sheet/solution.cpp
#include <vector>
#include <string>
#include <unordered_map>
#include <unordered_set>
#include <algorithm>
#include <climits>
#include <queue>
#include <stack>
#include <cmath>
#include <numeric>
#include <set>
#include <map>
using namespace std;

class Solution {
public:
    vector<string> cellsInRange(string s) {
        vector<string> result;
        for (char c = s[0]; c <= s[3]; c++) {
            for (char r = s[1]; r <= s[4]; r++) {
                result.push_back(string(1, c) + r);
            }
        }
        return result;
    }
};
